src/inc/til/rle.h

```diff
@@ -1042,18 +1042,6 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 #ifdef __WEX_COMMON_H__
 namespace WEX::TestExecution
 {
-    template<typename T, typename S, typename Container>
-    class VerifyOutputTraits<::til::basic_rle<T, S, Container>>
-    {
-        using rle_vector = ::til::basic_rle<T, S, Container>;
-
-    public:
-        static WEX::Common::NoThrowString ToString(const rle_vector& object)
-        {
-            return WEX::Common::NoThrowString(object.to_string().c_str());
-        }
-    };

```

Comment:
...yes. 😑
Let me know if you dislike this removal. `rle_vector` uses strstream like all the other classes and `TextAttribute` has no string-serializer. I just removed it because it was easier. I could however write a full `ToString` method here if we want to.

Reply:
yeah honestly I don't particularly care for all of these